src/main/java/uk/gov/justice/digital/delius/jpa/standard/entity/Contact.java

```diff
@@ -211,12 +221,20 @@ public boolean isRarActivity() {
 
         // the NSI is the primary component of RAR, so we check that first.
         // we also don't have to be careful with defaults here as the requirement and nsi foreign keys are mutually exclusive.
-        return Optional.ofNullable(getNsi())
+        return ofNullable(getNsi())
```

Comment:
Very minor, personally I prefer not to statically import Optional, a bit like if you have `List.of` it would be weird to just have `of("banana")`

Reply:
Fixed